### ISSUE TITLE
Visuelement, le menu langue a trop d'importance

### ISSUE THREAD

**Issue by zufrieden** [antistatique/athena-landingpage#1]
- [ ] par rapport au bouton de contact, le menu langue doit être d'un niveau inférieur

**Comment by guillaumeberry-zz** [comment]
Agreed! On a déjà parlé de ça avec @Yago mais Skeleton n'a que le bouton en composant :(
![agreed](https://cloud.githubusercontent.com/assets/1800253/7315422/8b4195a0-ea6d-11e4-991a-662c6eec04b8.png)
Un truc du genre peut-être ?

**Comment by zufrieden** [comment]
Oui, mais au niveau design. Le fameux dilem de la langue active. Fait 2-3
user test et demande : Quelle est la lange active ?
Il faut donc 3 états ;-) et du coup, inverser les couleurs n'est pas une
bonne idée !

On Fri, Apr 24, 2015 at 10:35 AM, Guillaume Berry notifications@github.com
wrote:

> Agreed! On a déjà parlé de ça avec @Yago https://github.com/Yago mais
> Skeleton n'a que le bouton en composant :(
> [image: agreed]
> https://cloud.githubusercontent.com/assets/1800253/7315422/8b4195a0-ea6d-11e4-991a-662c6eec04b8.png
> Un truc du genre peut-être ?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/antistatique/athena-landingpage/issues/1#issuecomment-95852900
> .

## 

---

Marc Friederich
marc@antistatique.net

Antistatique http://antistatique.net/
rue de Sébeillon 9b
1004 Lausanne
+41 21 623 63 03
www.antistatique.net
